addons/mail/static/src/owl/components/thread/thread.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.component.Thread" owl="1">
-        <div class="o_Thread" t-att-data-thread-local-id="props.threadLocalId">
+        <div class="o_Thread" t-att-data-thread-local-id="props.threadLocalId" t-att-class="{ 'o-is-temporary': storeProps.thread.isTemporary }">
```

Comment:
unused modifier

Reply:
I know it's not "top" but it's in fact used in a test to check that the thread is temporary

Reply:
In that case, I would prefer test to assert that thread with local id in `data-thread-local-id` is temporary in the store.
Basically, and roughly:

```js
const threadLocalId = document.querySelector(`.o_Chatter_thread`).dataset.threadLocalId;
const thread = this.env.store.state.threads[threadLocalId];
assert.ok(thread.isTemporary);
```